core/Memory_Model/Task/TaskEntity.h

```diff
@@ -34,11 +34,11 @@ class TaskEntity {
   void                                                      SetComplete();
 
  private:
-  Task                                                  task_;
-  TaskID                                                id_;
-  TaskID                                                parentID_;
-  bool                                                  complete_;
-  std::map<TaskID, std::weak_ptr<TaskEntity>>           subtasks_;
+  Task                                                      task_;
+  TaskID                                                    id_;
+  TaskID                                                    parentID_;
+  bool                                                      complete_;
+  std::map<TaskID, std::weak_ptr<TaskEntity>>               subtasks_;
 };
```

Comment:
Refactoring should be in separate PR, you're annigilyatornaya pushka

Reply:
oh, shit, i'm sorry ahhahah